set.mm

```diff
@@ -25929,11 +25929,24 @@ choice between (what we call) a "definitional form" where the shorter
   $}
 
   ${
-    $d y ph $.  $d x ps $.
+    $d y z ph $.  $d x z ps $.  $d x y $.
     cbvabv.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
     $( Rule used to change bound variables, using implicit substitution.
-       (Contributed by NM, 26-May-1999.) $)
+       (Contributed by NM, 26-May-1999.)  Let ` x ` , ` y ` be disjoint to
```

Comment:
Let --> Require.  I would write:
``Require ` x ` and ` y ` to be disjoint to...``
or
``Add a DV condtion on ` x , y ` to...``
(I'm not sure how we write "DV", though: maybe "dv" or maybe "disjoint variable").